src/main/java/com/smartcar/sdk/data/VehicleBattery.java

```diff
@@ -0,0 +1,69 @@
+package com.smartcar.sdk.data;
+
+/**
+ * POJO for Smartcar /battery endpoint
```

Comment:
Is this a POJO for the endpoint? Or is it just a POJO to represent a `VehicleBattery` reading